pages/instructorAssessmentInstances/instructorAssessmentInstancesClientJS.ejs

```diff
@@ -270,6 +270,12 @@ function detailsLinkSorter(valueA, valueB, rowA, rowB) {
     return compare;
 }
 
+function timeRemainingLimitSorter(valueA, valueB, rowA, rowB) {
+    // Closed assessments are listed first, followed by time limits
+    // ascending, followed by open without a time limit
+    return (rowA.open - rowB.open) || ((valueA ?? Infinity) - (valueB ?? Infinity));
```

Comment:
Subtracting booleans is neat, bordering on too clever 😅 this is a little more explicit:
```suggestion
    return (Number(rowA.open) - Number(rowB.open)) || ((valueA ?? Infinity) - (valueB ?? Infinity));
```